src/packages/@common/components/header/Header.svelte

```diff
@@ -294,6 +315,24 @@
       borderRounded={"2px"}
       position={"absolute"}
     />
+
+    {#if !isGuestUser}
+      <div class="ms-4 me-2">
+        <Tooltip
+          title="User Profile"
+          placement="bottom"
+          distance={20}
+          zIndex={5}
+          show={!showProfileModal}
+        >
+          <SidebarProfileModal
+            item={sidebarModalItem}
+            {onLogout}
+            bind:showProfileModal
+          />
+        </Tooltip>
+      </div>
+    {/if}
```

Comment:
create a separate component in header for profile

Reply:
Resolved